Tests/RxCocoaTests/DelegateProxyTest+UIKit.swift

```diff
@@ -265,6 +259,36 @@ final class UIScrollViewSubclass
     }
 }
 
+final class ExtendScrollViewDelegateProxy2
+    : RxScrollViewDelegateProxy
+, TestDelegateProtocol {
```

Comment:
In a lot of places there is `, XXX`. It would be great to properly align all of these places.